lib/network/api/transaction.go

```diff
@@ -56,10 +57,7 @@ func (api NetworkHandlerAPI) GetTransactionsHandler(w http.ResponseWriter, r *ht
 	prev := GetTransactionsHandlerPattern + "?" + options.SetReverse(true).Encode()
 	list := resource.NewResourceList(txs, self, next, prev)
 
-	if err := httputils.WriteJSON(w, 200, list); err != nil {
-		http.Error(w, "Error reading request body", http.StatusInternalServerError)
-		return
-	}
+	httputils.WriteJSON(w, 200, list)
```

Comment:
`MustWriteJSON`? or checking err nil?

Reply:
@Charleslee522 Let shows this pr desc.

Reply:
I think @Charleslee522 point was that you are using `WriteJSON` instead of `MustWriteJSON` here.

Reply:
It depends on client-side, If you have more error handling, you can use `WriteJSON`, If you are not, just 
 use `MustWriteJSON` as notice as PR`desc.

Reply:
So then, I think using `MustWriteJSON` is right in this part. But not now :)